beets/autotag/__init__.py

```diff
@@ -35,6 +35,41 @@
 # Global logger.
 log = logging.getLogger('beets')
 
+# metadata that is already hardcoded
+MISC_FIELDS = {
```

Comment:
This now does the *opposite* of what misc_fields used to do but it’s still using the old name. Let’s call this SPECIAL_FIELDS and use this comment:

> Most fields are applied directly using the same name as the field in the AlbumInfo or TrackInfo object. We handle a few fields specially, however, and avoid copying them directly to the object.